docs/src/pages/components/about-the-lab/about-the-lab.md

```diff
@@ -2,6 +2,18 @@
 
 <p class="description">This package hosts the incubator components that are not yet ready to move to the core.</p>
 
+The main difference between the lab and the core is how the components are versioned. Having the lab within Material UI allows us to release breaking changes when necessary while the core package follows a [slower-moving policy](https://material-ui.com/versions/#release-frequency).
+
+As our users use and test the components and report issues, we learn more about the design flaws of our components: lacking features, a11y, bugs, API, etc. The older and more used a component is, the less likely it is that we will find new flaws and subsequently, need to introduce breaking changes.
```

Comment:
I wonder, shouldn't we remove the usage of "our"/"us"? Here is how it would look like in this paragraph.

```suggestion
As developers use and test the components and report issues, the maintainers learn more about the design flaws of the components: lacking features, accessibility, bugs, API, etc. The older and more used a component is, the less likely it is that new flaws will be found and subsequently, need to introduce breaking changes.
```